bot/models/migrations/004_up__levelling.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE IF NOT EXISTS levelling_users
 CREATE TABLE IF NOT EXISTS custom_roles
 (
     id          BIGINT PRIMARY KEY DEFAULT create_snowflake(),
+    user_id     BIGINT NOT NULL,
```

Comment:
After a PR has been merged you don't edit a migration. Migrations are run when merged and they are only run once, that means that this change will not be included in the prod database.

You need to create a new migration to alter the table.
This field should also be nullable right..